src/main/java/net/pms/dlna/RootFolder.java

```diff
@@ -726,7 +728,12 @@ private static DLNAResource getiPhotoFolder() {
 			} catch (XmlParseException | URISyntaxException | IOException e) {
 				LOGGER.error("Something went wrong with the iPhoto Library scan: ", e);
 			} finally {
-				IOUtils.closeQuietly(inputStream);
+				try {
+					inputStream.close();
```

Comment:
I agree with this change, but instead of doing it like this, let's use try-with-resources so we can remove the `finally` block completely, as this answer suggests https://stackoverflow.com/a/49251654/2049714